src/screens/MemberDetail/MemberDetail.tsx

```
@@ -292,15 +293,30 @@ const MemberDetail: React.FC<MemberDetailProps> = ({ id }): JSX.Element => {
-                        name={formState.name}
-                        alt="User Image"
-                        size={60}
-                        dataTestId="profile-picture"
-                        radius={150}
-                      />
+                      <div
+                        onClick={() => fileInputRef.current?.click()}
+                        data-testid="uploadImageBtn"
+                        style={{ cursor: 'pointer', fontSize: '1.2rem' }}
+                        title="Edit profile picture"
+                        role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)